src/everything/everything.ts

```diff
@@ -459,6 +484,11 @@ export const createServer = () => {
           "Returns a resource reference that can be used by MCP clients",
         inputSchema: zodToJsonSchema(GetResourceReferenceSchema) as ToolInput,
       },
+      {
+        name: ToolName.ELICITATION_DEMO,
+        description: "Demonstrates the elicitation feature",
```

Comment:
Maybe add a little more to the description?

```suggestion
        description: "Demonstrates the Elicitation feature by asking the user to provide information about their favorite color, number, and pets.",
```